src/components/Cards/styles/HeaderWrapper.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import themeGet from 'extensions/themeGet';
+
+const HeaderWrapper = styled.div`
+  color: ${themeGet('colors.background.default')};
+  font-size: 1.2em;
+  font-weight: 500;
+  text-transform: uppercase;
+  background-image: url(${props => props.image || 'none'});
```

Comment:
If I understand correctly, this component will contain a white-colored header. To avoid that header becoming invisible if an image is not provided, a background color could also be specified. Maybe `colors.primary.dark`.

Reply:
This is a really good idea